src/OrchardCore.Modules/OrchardCore.Menu/Models/ContentPickerMenuItemPart.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using OrchardCore.ContentManagement;
+
+namespace OrchardCore.Menu.Models
+{
+    public class ContentPickerMenuItemPart : ContentPart
+    {
+        /// <summary>
+        /// The name of the link
+        /// </summary>
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Content Picker ItemIds
+        /// </summary>
+        [BindProperty(Name = "ContentItemIds")]
+        public string ContentItemId { get; set; }
```

Comment:
this binding seems uneeded?

Reply:
Ah, ok, I looked further, and figure out what you are doing here :)

So you are storing the `ContentItemId` on the `ContentPickerMenuItemPart`, but it is also stored on the `ContentPickerField` that you have added to the part definition.

So maybe we don't need to be storing it twice, and you could just remove this field completely, and access the content picker value with the normal accessor `part.ContentItem.Content.ContentPickerMenuPart.ContentPickerField.ContentItemIds[0]`

It is longer, but then we are not storing the value twice, and the content picker field driver handles the json ids from the vue component properly